port/zos390/omrvmem.c

```diff
@@ -1179,7 +1185,7 @@ reserve4KPages(struct OMRPortLibrary *portLibrary, struct J9PortVmemIdentifier *
 #if defined(OMR_ENV_DATA64)
 	} else {
 		/* use the 2to32GArea */
-		baseAddress = reservePagesAboveBar(portLibrary, identifier, params->mode, params->byteAmount, FOUR_K, OMRPORT_VMEM_PAGE_FLAG_PAGEABLE, params->options, category);
+		baseAddress = reservePagesAboveBar(portLibrary, identifier, params, category);
```

Comment:
I would double check here is `params->pageSize` set to  `FOUR_K` and `params->pageFlags` to OMRPORT_VMEM_PAGE_FLAG_PAGEABLE` here. Otherwise you might need to set them this way.